plugins/.gitignore

```diff
@@ -23,5 +23,9 @@
 node_modules/
 src/core/**/*.js
 npm-debug.log
+xcuserdata
+/platforms/
+/plugins/
+optimove.json
```

Comment:
Do these need adding to the plugin ignore? I'm not 100% sure, I think this already exists at the root repository level, and there are no Xcode projects inside the plugins dir itself.

Reply:
done